scripts/deploy-federation.sh

```diff
@@ -93,6 +93,7 @@ if [[ ! "${USE_LATEST}" ]]; then
     cp ${base_dir}/bin/controller-manager ${dockerfile_dir}/controller-manager
   fi
   docker build ${dockerfile_dir} -t "${IMAGE_NAME}"
+  docker push "${IMAGE_NAME}"
```

Comment:
(No action required) Hmm, I didn't see this additional complexity pre-merge.  @irfanurrehman is there a reason this script doesn't reuse the makefile?

Reply:
The script predates Makefile; I guess it makes sense to update all scripts to use Makefile now. Created https://github.com/kubernetes-sigs/federation-v2/issues/182 for the same.

Reply:
Yes, we need to migrate to the use of the Makefile throughout the repo e.g. docs and scripts.